plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotTableProperties.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.pinot;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.spi.session.PropertyMetadata;
+
+import java.util.List;
+
+import static io.trino.spi.session.PropertyMetadata.stringProperty;
+
+public class PinotTableProperties
+{
+    public static final String PINOT_COLUMN_NAME_PROPERTY = "pinot_column_name";
+    private final List<PropertyMetadata<?>> columnProperties;
+
+    public PinotTableProperties()
+    {
+        columnProperties = ImmutableList.<PropertyMetadata<?>>builder()
+                .add(stringProperty(PINOT_COLUMN_NAME_PROPERTY, "Pinot columnName", null, false))
```

Comment:
I'm not sure if we want to expose this property to users. Can we make it hidden?

Reply:
Sure, but there might be some value, as it will show the actual pinot column name if it contains non lower case characters. I have a pr (need to update) for create table which uses this and many more properties. Should I leave it hidden for now and update in the future when it becomes more relevant? Or do you think it's worth exposing now bcz of non-lower case characters in tables?

Reply:
I think it would be better to hide for now because exposing hidden table property later wouldn't break compatibilities.